web-client/src/components/InventoryTable.tsx

```diff
@@ -0,0 +1,124 @@
+import { useState } from 'react';
+import { Trash2 } from "lucide-react";
+import { Button } from "@/components/ui/button";
+import DeleteConfirmationDialog from "@/components/DeleteConfirmationDialog";
+import { useDeleteSockMutation } from "@/api/inventory/queries";
+
+interface Variant {
+    id: number;
+    size: string;
+    price: number;
+    quantity: number;
+}
+
+interface Sock {
+    id: number;
+    name: string;
+    variants: Variant[];
+    imageUrl: string;
+}
+
+interface InventoryTableProps {
+    socks: Sock[];
+    onRowClick: (sockId: string) => void;
```

Comment:
`sockId` should be a `number` I believe

Reply:
Done in this commit:
https://github.com/sockify/sockify/commit/e62bec362a07dd99e36795536bd5dc70b6bd12b9